core/kazoo_media/src/kazoo_media_maintenance.erl

```diff
@@ -239,17 +240,19 @@ maybe_retry_upload(ID, AttachmentName, Contents, Options, Retries) ->
 
 -spec refresh() -> 'ok'.
 refresh() ->
-    Req = [{<<"Database">>, ?KZ_MEDIA_DB}
-          ,{<<"Classification">>, kz_datamgr:db_classification(?KZ_MEDIA_DB)}
-          ,{<<"Action">>, <<"refresh_views">>}
-           | kz_api:default_headers(?APP_NAME, ?APP_VERSION)
-          ],
-    {'ok', _} = kz_amqp_worker:call(Req
-                                   ,fun kapi_maintenance:publish_req/1
-                                   ,fun kapi_maintenance:resp_v/1
-                                   ),
+    case kz_datamgr:db_exists(?KZ_MEDIA_DB) of
+        'false' ->
+            Result = kz_datamgr:db_create(?KZ_MEDIA_DB),
+            lager:debug("~s database is created: ~p", [?KZ_MEDIA_DB, Result]);
+        'true' -> 'ok'
+    end,
+    _ = kapps_maintenance:refresh(?KZ_MEDIA_DB),
     'ok'.
 
+-spec register_views() -> 'ok'.
+register_views() ->
+    kz_datamgr:register_views_from_folder('kazoo_media').
```

Comment:
this is not being called at startup

Reply:
`kazoo_media_app` is call `kapps_maintenance:bind_and_register_views` which doing the bind and calling register function.

Reply:
`true` and this is one of the reasons that i would like to see the `register_views` binding in a separate pull request. this way, you're turning a `utility` into a `must use` which i don't like.
on `app` startup
1) create any DB specific to the `app`
2) register views specific to the `app`
3) do other bindings

```
kapps_maintenance:bind_and_register_views('kazoo_media', 'kazoo_media_maintenance', 'register_views'),
kapps_maintenance:bind('migrate', 'kazoo_media_maintenance', 'migrate'),
kazoo_media_maintenance:refresh()
```
should be

```
kazoo_media_maintenance:init_dbs()
kapps_maintenance:bind('register_views', 'kazoo_media_maintenance', 'register_views'),
kapps_maintenance:bind('migrate', 'kazoo_media_maintenance', 'migrate'),

```
in kazoo_media_maintenance
```
init_dbs() ->
    _ = kz_datamgr:db_create(?KZ_MEDIA),
    _ = register_views().
```

this should be for all apps, i understand that `kapps_maintenance:bind_and_register_views` is like `killing 2 birds with one stone` but  will keep things simpler